pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller_suitetest.go

```diff
@@ -152,7 +152,8 @@ var _ = BeforeSuite(func(done Done) {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
+		CRDDirectoryPaths:     []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
+		ErrorIfCRDPathMissing: true,
```

Comment:
What happens if we create a controller without CRD? 

> $ kubebuilder create api --group ship --version v1beta1 --kind Frigate
> Create Resource [y/n]
> n
> Create Controller [y/n]
> y

Reply:
Just to explain a bit more what Camila is asking, controllers without a CRD can be used for builtin resources for example, or for resources that may be defined by other frameworks. So, technically, we could have a controller without its resource. So imagine we want to create a controller for `v1.Namespace` with no CRD (as the resource is already defined by k8s). Will this flag make the suite fail? In that case we need to set it only in the case we are not deploying a controller for an existing resource.

Reply:
Also see: https://github.com/kubernetes-sigs/kubebuilder/issues/1767
IMO we need to provide the fix first to be able to check this one.